streams/src/main/java/org/apache/kafka/streams/processor/TaskMetadata.java

```diff
@@ -68,17 +61,8 @@ public TaskMetadata(final TaskId taskId,
     /**
      * @return the basic task metadata such as subtopology and partition id
      */
-    public TaskId getTaskId() {
```

Comment:
Same here, this reverts to the state prior KIP-740. Users are anyway discouraged to use this class.